storage_service/locations/models/space.py

```diff
@@ -445,7 +445,7 @@ def move_rsync(self, source, destination, try_mv_local=False):
 
         # Rsync file over
         # TODO Do this asyncronously, with restarting failed attempts
-        command = ['rsync', '-t', '-O', '--protect-args', '-vv', '--chmod=ugo+rw', '-r', source, destination]
+        command = ['rsync', '-t', '-O', '--protect-args', '-vv', '--chmod=ugo+rw,Da+x', '--perms', '-r', source, destination]
```

Comment:
@jhsimpson you have an explanation of what the `D` stands for here: https://explainshell.com/explain?cmd=rsync+-p+--chmod%3D%2Brwx+-e+%22ssh+-i+userserver.pem%22+--copy-unsafe-links+-rz+user%40server%3A~%2F+%2Fdb_backups%2Fwww%2F
It does the same as `(upper) X`, but I wasn't aware of it ^^

@sevein you're right, `--perms` would only be useful for already existing files, which shouldn't be the case here.